signals.c

```diff
@@ -46,16 +45,11 @@ void signals_update_force()
   // ensure that the value being used doesn't change during 
   // during the function call.
   uint16_t raw_value = raw_adc_readings[FORCE];
-  
+
   // Index in signal buffer
   static const uint8_t n = N_FILTER - 1;
 
-  // Make sure that the offset doesn't make the sample negative.
-  if (signals.force_offset > raw_value) {
-    X_BUF(n) = 0;
-  } else {
-    X_BUF(n) = raw_value - signals.force_offset;
-  }
+  X_BUF(n) = raw_value;
```

Comment:
No wait, it's not necessary. We cannot get a value below 0 anymore. raw_value is the value read from the ADC which will alway be positive.

The subtraction of the offset made it possible for the value to be negative, but it's not longer the case.